assets/css/tw.css

```diff
@@ -183,6 +183,12 @@
 .content-aside ol > li + li {
   @apply mt-3;
 }
+.spl-input {
+  @apply w-full rounded border border-s-4 py-2 pl-4 pr-7 font-sans outline-none ring-2 ring-transparent focus:ring-yellow;
+}
+.spl-select {
+  @apply block w-full appearance-none rounded border border-s-4 bg-white py-2 pl-4 pr-7 font-sans text-s-6 shadow outline-none ring-2 ring-transparent focus:ring-yellow active:ring-yellow-6;
```

Comment:
Select has a shadow but input doesn't. Either way is fine, but they should be the same.